mock/provider.go

```diff
@@ -0,0 +1,18 @@
+package mock
+
+import (
+	"context"
+
+	"github.com/influxdata/flux/dependencies/influxdb"
+)
+
+type MockProvider struct {
```

Comment:
Can you rename this and the file `InfluxDBProvider` and `influxdb_provider.go` respectively?